content/posts/how-to-use-playwright-in-cucumberjs/index.md

```diff
@@ -0,0 +1,157 @@
++++
+date = ""
+author = "nploi"
+description = "How to use Playwright in cucumberjs"
+title = "How to use Playwright in cucumberjs"
+categories = ["e2e test"]
+tags = ["k8s", "playwright", "cucumberjs"]
+slug = "how-to-use-playwright-in-cucumberjs"
++++
+
+This tutorial helps you run automate your test using Playwright in Cucumber.
+
```

Comment:
```suggestion
This tutorial helps you run automate your test using Playwright in Cucumber.
```suggestion
This tutorial helps you run web automation tests by using Playwright in Cucumber.

```
```